packages/paste-website/src/components/StatusLegend.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import {Card} from '@twilio-paste/card';
+import {Heading} from '@twilio-paste/heading';
+import {Text} from '@twilio-paste/text';
+
+export const StatusLegend: React.FC<{}> = () => (
+  <Card>
+    <Heading as="h4" variant="heading40">
+      Component Status Details
+    </Heading>
+    <Text as="p" marginBottom="space20">
+      <strong>Alpha</strong>: Component is a candidate for testing in products with guidance from the design systems
+      team. If {"you 're"} planning on using one of these components, check in with the Paste team first through{' '}
```

Comment:
I'm not a fan of how dense `Alpha` is.  Would it be possible to omit the `If you're...` sentence and add a little blurb to the bottom of the card about reaching out about concerns and assistance instead?  I think it would look more visually balanced this way.

Reply:
Revised this based off the new doc from @SiTaggart